src/components/AsyncTable/AsyncTable.module.scss

```diff
@@ -38,4 +38,10 @@
   grid-row-start: 1;
   list-style: none;
   grid-gap: 0.875rem 0.75rem;
-}
\ No newline at end of file
+}
+
+.Empty {
+  margin-top: 2rem;
+  text-align: center;
+  font-style: italic;
```

Comment:
we could probs add font-style to variables. Unless we're waiting for theme engine stuff

Reply:
Copied from zDaos table, but I think we don't need `italic` font style